bigtable/client/row_reader.cc

```diff
@@ -94,6 +94,7 @@ void RowReader::MakeRequest() {
     request.set_rows_limit(rows_limit_ - rows_count_);
   }
 
+  context_ = absl::make_unique<grpc::ClientContext>();
```

Comment:
I totally failed to see the warning in the docs to not reuse the context. Sorry, and thanks for debugging this!

Reply:
No problem.